README.md

```diff
@@ -116,8 +116,145 @@ An example layer file in the path `/layers/ken-health-sites.json`
 }
 
 ```
+Gisida supports various data sources. These include: <br>
```

Comment:
Let's make these have markdown headers for better scanability, like so:
```
## Data Sources
Gisida supports various data sources. These include:
```

which looks like: 
## Data Sources
Gisida supports various data sources. These include: